charts/pulsar/values.yaml

```diff
@@ -228,29 +228,48 @@ tls:
     keySize: 4096
     keyAlgorithm: RSA
     keyEncoding: PKCS8
+    caSecretName:
   # settings for generating certs for proxy
   proxy:
     enabled: false
     cert_name: tls-proxy
     createCert: true  # set to false if you want to use an existing certificate
+    # specify name of secret contain certificate if using pre-generated certificate
+    certSecretName:
+    untrustedCa: true
   # settings for generating certs for broker
   broker:
     enabled: false
     cert_name: tls-broker
+    # specify name of secret contain certificate if using pre-generated certificate
+    certSecretName:
+    untrustedCa: true
   # settings for generating certs for bookies
   bookie:
     enabled: false
     cert_name: tls-bookie
+    # specify name of secret contain certificate if using pre-generated certificate
+    certSecretName:
+    untrustedCa: true
   # settings for generating certs for zookeeper
   zookeeper:
     enabled: false
     cert_name: tls-zookeeper
+    # specify name of secret contain certificate if using pre-generated certificate
+    certSecretName:
+    untrustedCa: true
```

Comment:
I don't see this key `Values.zookeeper.untrustedCa` being referenced in the templates.

Reply:
this comment is fixed by https://github.com/apache/pulsar-helm-chart/pull/559/commits/736dbf8b7523cfa161601a554a9777b31b468ca3 commit